mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs

```diff
@@ -2,10 +2,8 @@
 using System.Collections.Generic;
 using System.Text;
 using System.Reflection;
-
-#if ENABLE_CECIL
```

Comment:
The ENABLE_CECIL stuff was added by:
https://github.com/mono/mono/commit/83d63bf1818d0f151f5ab06a61422caab445f4a7
Maybe its still needed ?

Reply:
Yes, I talked to @jbevain and it's needed. Thanks